plugins/out_stdout/stdout.c

```diff
@@ -239,17 +239,15 @@ static void cb_stdout_flush(struct flb_event_chunk *event_chunk,
 
             printf("]\n");
         }
-
-        if (result != FLB_EVENT_DECODER_ERROR_INSUFFICIENT_DATA) {
-            flb_plg_error(ctx->ins, "decoder error : %d", result);
-        }
+        result = flb_log_event_decoder_get_decode_result(&log_decoder);
 
         flb_log_event_decoder_destroy(&log_decoder);
     }
 
     fflush(stdout);
 
     if (result != FLB_EVENT_DECODER_SUCCESS) {
+        flb_plg_error(ctx->ins, "decoder error : %d", result);
```

Comment:
Change the message to `Log event decoder error : %d`

Reply:
I modified the message.